Changelog.md

```diff
@@ -1,4 +1,5 @@
 # Next
+- Updated minimum version of `ReactiveSwift` to 3.0.
```

Comment:
We're almost there 😄 We need a *Breaking change* alias before the entry, and a column (probably `Changed` is the most fitting):
```
# Next
### Changed
- **Breaking Change** Updated minimum version of `ReactiveSwift` to 3.0.
```

Reply:
done :)